oshi-core/src/main/java/oshi/util/Util.java

```diff
@@ -65,4 +71,35 @@ public static void sleepAfter(long startTime, long ms) {
             sleep(until - now);
         }
     }
+
+    /**
+     * Generates a licence key based upon the processor serial number, vendor,
```

Comment:
Can you add a cautionary note to this javadoc that it may not be completely unique and can easily be defeated, or some other caveats from the StackOverflow thread?

Reply:
I was wondering though, if we share the actual value of our serial number instead of hash value will it be better? As Serial numbers are unique whereas hash value of the same may not be.

Reply:
Valid question.  (Same question could apply for "number of processors" which could just be left as an int.  I suspect the hash is unique enough for fingerprinting (and may not be accessible on many systems without root access anyway.)  My thought was to make the format easily parseable for comparison purposes, however, which would include transforming the string in such a manner that it's guaranteed not to contain the delimiter.   Which would you rather use as a user?  I'm not hard set one way or the other.

Reply:
True, I will mention the general warning of our key may not be unique.